client/src/components/Login.js

```diff
@@ -1,12 +1,52 @@
 import React from "react";
+import { axiosWithAuth } from "../utils/axiosWithAuth";
+
+const Login = props => {
+  const [loginForm, setLoginForm] = React.useState({
+    username: "",
+    password: ""
+  });
+
+  const handleChanges = e => {
+    setLoginForm({ ...loginForm, [e.target.name]: e.target.value });
+  };
+
+  const login = e => {
+    e.preventDefault();
+    axiosWithAuth()
+      .post("/api/login", loginForm)
```

Comment:
The `axiosWithAuth()` component you've made has a baseURL of `http://localhost:5000/api`.

With this endpoint, your final URL will have `/api` typed twice:
```
http://localhost:5000/api/api/login
```

You should be good to remove the `/api` from this axios call 👍